common/constants.js

```diff
@@ -271,6 +274,12 @@ module.exports = Object.freeze({
     API_VERSION: 'v1alpha1',
     CRD_RESOURCE_GROUP: 'apiextensions.k8s.io',
     PATCH_CONTENT_TYPE: 'application/merge-patch+json',
+    CONFIG_MAP: {
+      API_VERSION: 'v1',
+      RESOURCE_TYPE: 'configmaps',
+      RESOURCE_NAME: 'sfconfig',
```

Comment:
shouldnt this constant go into CONFIG? Because config_map is generic and one can create more than one config map with a different resource name. So i would move this to `CONFIG.SF_CONFIG`

Reply:
done!